README.md

```diff
@@ -135,6 +135,16 @@ The `anchorDirection` prop indicates whether the calendar is anchored to the rig
   withFullScreenPortal: PropTypes.bool
 ```
 
+The `bottomPanel` prop accepts a React element that will be displayed below `DayPicker`.
+```
+  bottomPanel: PropTypes.element
+```
+
+The `portalMarginTop` prop sets the space between `DateRangePickerInput` and the `DayPicker` popout.
```

Comment:
this should not be named `portalMarginTop` because the `DayPicker` is not necessarily in a portal (and in fact this margin does not affect the case where it is in a portal)